packages/discordx/src/Client.ts

```diff
@@ -553,6 +556,7 @@ export class Client extends ClientJS {
 
     // filter application command to update
 
+    const commandsWithChanges: ApplicationCommand[] = [];
```

Comment:
Could do with a better name for this. Basically I need to have access to the original commands if the `update` option is disabled.